torchdata/datapipes/iter/load/aisio.py

```diff
@@ -17,19 +15,37 @@
     from aistore.pytorch.utils import parse_url, unparse_url
 
     HAS_AIS = True
+
 except ImportError:
     HAS_AIS = False
 
+try:
+    import aistore
+    from packaging.version import parse as parse_version
+
+    AIS_VERSION_CHECK = parse_version(aistore.__version__) >= parse_version("1.0.2")
```

Comment:
```suggestion
    AIS_VERSION_CHECK = parse_version(aistore.__version__) == parse_version("1.0.2")
```
nit: to be consistent with `requirements_aistore.txt`. Let me know if there is a reason to be different.